plugins/inputs/prometheus_scraper/metrics_type_handler.go

```diff
@@ -55,6 +55,10 @@ func (pm *PrometheusMetric) isSummary() bool {
 	return pm.metricType == textparse.MetricTypeSummary
 }
 
+func (pm *PrometheusMetric) isUnknown() bool {
```

Comment:
I see that this function is called [here](https://github.com/aws/amazon-cloudwatch-agent/pull/407/files#diff-7afaf53868de6a1aec096eab2439312727c4ca612d2f54eb5ebed758c1feac8aR47) which happens to be in the same package so it is accessible. Should this function be made public?

Reply:
Are there any plugins besides `Prometheus` using these types? If not, I don't see any reason to make them public though.